### DISCUSSION TITLE
File Request

### DISCUSSION THREAD

**Discussion by the-Jack-D** [diced/zipline#641] [Questions]
Box has a feature called File Request where you can create a link and provide it to someone, and they can use it to send a file/files without authenticating. Is this possible with Zipline?

For example, let's say I need a friend or someone to send me a large file. Rather than have them create an account, I could just send them a link and they could drop the files there and then they would be available to me. They wouldn't be shared back out to that person or anyone else.. incoming only, unique link, no authentication.

**Comment by SaiCode-OTH** [comment]
Defininitly need this - the only important feature missing for me.

Perhaps also allow multiple files or just one, with a time expiration.

**Comment by SaiCode-OTH** [comment]
@diced perhaps can you change the category or do i have to create a duplicate in ideas?

**Comment by SilverServer34** [comment]
I’m looking for a replacement for Palmr, and Zipline ticks almost all my boxes,  except for this feature.

What I really need is a simple “incoming only” upload link that I can share with someone so they can send me files without creating an account. No access to other files, no dashboard, just a one-way upload page.

I already use ProjectSend when I need account-based workflows, so having to create users in Zipline just for occasional uploads feels like too much overhead.

If Zipline ever supported anonymous upload links (similar to Box File Request or Palmr’s upload links), it would perfectly complete my setup.
